src/App.tsx

```diff
@@ -75,12 +75,16 @@ const App: React.FC = () => {
   }
 
   if (loadingState === 'failed') {
-    return (
-      <Result
-        status="warning"
-        title="Failed to load the initial data. Check your console."
-      />
-    );
+    if (config?.path?.auth?.login) {
+      window.location.href = config.path.auth.login;
```

Comment:
Untested, but we might want to check if we could make use of the `isSessionValid` interface to check if the user is actually logged in before we redirect to the login page.